plugins/inputs/gnmi/gnmi_test.go

```diff
@@ -1011,193 +1018,319 @@ func TestRedial(t *testing.T) {
-			insertOps: []insertOp{
-				{
-					keys:  []*gnmiLib.PathElem{protocolDoubleKey},
-					name:  "doubleKey",
-					value: &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_JsonVal{JsonVal: []byte("{}")}},
-				},
-			},
-			expected: &tagNode{
-				tagStore: map[string][]*tagNode{
-					"protocol": {
-						{
-							elem:    protocolDoubleKey,
-							value:   &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_JsonVal{JsonVal: []byte("{}")}},
-							tagName: "doubleKey",
-						},
-					},
-				},
-			},
-		},
-		{
-			name: "multi elem unrelated insert",
-			insertOps: []insertOp{
-				{
-					keys:  []*gnmiLib.PathElem{interfaceElemSingleKey},
-					name:  "intf_desc",
-					value: &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "mgmt"}},
-				},
-				{
-					keys:  []*gnmiLib.PathElem{networkInstanceSingleKey, protocolDoubleKey, neighborSingleKey},
-					name:  "bgp_neigh_desc",
-					value: &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "example-neighbor"}},
-				},
-			},
-			expected: &tagNode{
-				tagStore: map[string][]*tagNode{
-					"interface": {
-						{
-							elem:    interfaceElemSingleKey,
-							value:   &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "mgmt"}},
-							tagName: "intf_desc",
-						},
-					},
-					"network-instance": {
-						{
-							elem: networkInstanceSingleKey,
-							tagStore: map[string][]*tagNode{
-								"protocol": {
-									{
-										elem: protocolDoubleKey,
-										tagStore: map[string][]*tagNode{
-											"neighbor": {
-												{
-													elem:    neighborSingleKey,
-													value:   &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "example-neighbor"}},
-													tagName: "bgp_neigh_desc",
-												},
-											},
-										},
-									},
-								},
-							},
-						},
-					},
-				},
-			},
-		},
-		{
-			name: "values at multiple levels",
-			insertOps: []insertOp{
-				{
-					keys:  []*gnmiLib.PathElem{networkInstanceSingleKey},
-					name:  "vrf_stuff",
-					value: &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "foo"}},
-				},
-				{
-					keys:  []*gnmiLib.PathElem{networkInstanceSingleKey, protocolDoubleKey},
-					name:  "protocol_stuff",
-					value: &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "bar"}},
-				},
-			},
-			expected: &tagNode{
-				tagStore: map[string][]*tagNode{
-					"network-instance": {
-						{
-							elem:    networkInstanceSingleKey,
-							value:   &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "foo"}},
-							tagName: "vrf_stuff",
-							tagStore: map[string][]*tagNode{
-								"protocol": {
-									{
-										elem:    protocolDoubleKey,
-										value:   &gnmiLib.TypedValue{Value: &gnmiLib.TypedValue_StringVal{StringVal: "bar"}},
-										tagName: "protocol_stuff",
-									},
-								},
-							},
-						},
+// func TestTagNode(t *testing.T) {
```

Comment:
If you are going to comment this all out, can you remove it?

Reply:
A bunch of the latest builds were for testing to make this stuff work, so I would expect that there will be a bunch of stuff in there to be cleaned up by @srebhan